### PR TITLE
OpenStack-Permissions-01

### PR DESCRIPTION
**Pull request by resmaa**
1. CLI Gateway Creation
2. Permission set OpenStack

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/415#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: packages/docs/content/04-clusters/1-new-clusters.md [hunk 1]
```diff
@@ -2014,86 +2268,97 @@ By default, 4GB of memory is allocated for private gateways. Please ensure that
 * Please ensure the following prerequisites are met in your OpenStack environment:
     * An instance with an Ubuntu 20.x or higher
```
Comment (arvind-sys) [inline packages/docs/content/04-clusters/1-new-clusters.md:2269]: Not Necessary.
* Machine having Docker Engine Running

File: packages/docs/content/04-clusters/1-new-clusters.md [hunk 2]
```diff
@@ -2014,86 +2268,97 @@ By default, 4GB of memory is allocated for private gateways. Please ensure that
...
-|svc_ip_range (optional)| CIDR notation IP range from which to assign service cluster IPs (This must not overlap with any IP ranges assigned to nodes or pods)|
+* As a tenant administrator, navigate to the Private Cloud Gateway page under admin settings and invoke the dialogue to create a new private cloud gateway in Open Stack cloud.
+* Make a note of the unique pairing code which is the unique authentication code generated for the installer instance.
+
+### Detailed Instructions:
+
+#### Step 1: Run the docker command to install the gateway as below:
+
+``` json
+docker run --rm  \--net=host  \-it  \-v /var/run/docker.sock:/var/run/docker.sock \-v /tmp:/opt/spectrocloud \gcr.io/spectro-images-public/release/spectro-installer:v1.0.0 \pcg  \openstack-pcg \OpenStack \-w=/opt/spectrocloud/ \-o=true
```
Comment (arvind-sys) [inline packages/docs/content/04-clusters/1-new-clusters.md:2279]: Image: gcr.io/spectro-images-public/release/spectro-installer:v1.0.1

File: packages/docs/content/04-clusters/1-new-clusters.md [hunk 3]
```diff
@@ -2014,86 +2268,97 @@ By default, 4GB of memory is allocated for private gateways. Please ensure that
...
 <InfoBox>
 This step does not apply to Enterprise version users.
 </InfoBox>	
 
-To deploy the Gateway installer:
-
-* Download the utility file to /tmp folder
+### To deploy the Gateway installer:
 
+* Go to /temp/utility-folder and view the configuration yaml file.
```
Comment (arvind-sys) [inline packages/docs/content/04-clusters/1-new-clusters.md:2337]: /tmp/install-pcg-openstack-pcg-<unique-ID>

File: packages/docs/content/04-clusters/1-new-clusters.md [hunk 4]
```diff
@@ -2014,86 +2268,97 @@ By default, 4GB of memory is allocated for private gateways. Please ensure that
...
-  Sample: wget https://spectro-cli.s3.amazonaws.com/pcg/pcg_installer.bin
-
+cd /tmp 
+cd install-pcg-openstack-pcg-20210726103558 
+vi pcg.yaml
 ``` 
-* Non-Proxy:
-  Initiate deployment using the command below with pairing value obtained from tenant console, console url and gateway name of customer choice.
+* Update or verify the installer name and make a note of the installer name for the deployment.
+* Run the deployment command as:
```
Comment (arvind-sys) [inline packages/docs/content/04-clusters/1-new-clusters.md:2345]: One Step Above
* Move the PCG.yaml file to /tmp folder for easy Execution

File: packages/docs/content/04-clusters/1-new-clusters.md [hunk 5]
```diff
@@ -2014,86 +2268,97 @@ By default, 4GB of memory is allocated for private gateways. Please ensure that
...
-|SSH Key Name| The key, useful for troubleshooting purposes (Recommended)|Enables SSH access to the VM as 'ubuntu' user|
-|Placement|Dynamic/Static |  |
-|Node CIDR|Optional - IP range exclusive to Nodes|This range should be different to prevent an overlap with your network CIDR|
-|DNS Name Server (optional)|8.8.8.8 default|A system by which Internet domain name-to-address and address-to-name resolutions are determined|
-|Availability Zones|Make the choices from the list. |Logical abstraction for partitioning an OpenStack cloud for users|
-|Flavor| Select from the drop-down list|Defines the compute, memory, and storage capacity of nova computing instances|
-|Number of nodes|1 or 3 | |
+* Navigate back to the Private Cloud Gateway page in tenant console under settings.
+* The deployment progress can be viewed in the tenant console. This will take few minutes
+* Once the deployment is  completed, set the number of nodes as either 1 or 3.
```
Comment (arvind-sys) [inline packages/docs/content/04-clusters/1-new-clusters.md:2360]: Not required. As we have already set. Remove this point "Once the deployment is completed, set the number of nodes as either 1 or 3."

**Review by arvind-sys — CHANGES_REQUESTED** [review]

**Pull request closed without merging**